ingest/importer/submission/entity_map.py

```diff
@@ -0,0 +1,120 @@
+from typing import List
+
+from ingest.importer.submission.entity import Entity
+
+
+class EntityMap(object):
+
+    def __init__(self, *entities):
+        self.entities_dict_by_type = {}
+        if entities is not None:
+            for entity in entities:
+                self.add_entity(entity)
+
+    @staticmethod
+    def load(entity_json):
```

Comment:
This function is very complex.

Reply:
Won't touch this file for now.

Reply:
This was also moved to its own file. But I haven't touched this function.

Reply:
Refactored.